main.go

```diff
@@ -90,6 +103,15 @@ It can be used in a curl-like fashion when benchmarking features are disabled.
 		return opts, errExit
 	}
 
+	// Read defaults from ~/.yab.ini
+	if user, err := user.Current(); err != nil {
```

Comment:
instead of using else, suggest reduce nesting here,
```
user, err := user.Current()
if err != nil {
  // fatal
}

[...]
homeDir := user.HomeDir 
```